tests/s3/versioning/test_limits_tagging_versioning.py

```diff
@@ -221,3 +221,155 @@ def test_tag_key_min_41278(self, versioning_status):
                     self.object_name, latest_ver, get_tag)
         LOGGER.info("Completed: Test minimum key length of a tag for a versioned object - "
                     "1 character")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.tags("TEST-41279")
+    @pytest.mark.parametrize("versioning_status", ["Enabled", "Suspended"])
+    def test_tag_value_limit_41279(self, versioning_status):
+        """
+        Test maximum value length of a tag for a versioned object - 256 Unicode characters
+        """
+        LOGGER.info("Started: Test maximum value length of a tag for a versioned object - "
+                    "256 Unicode characters")
+        LOGGER.info("Step 1: Perform PUT Bucket versioning with status as %s on %s",
+                    versioning_status, self.bucket_name)
+        resp = self.s3_ver_obj.put_bucket_versioning(bucket_name=self.bucket_name,
+                                                     status=versioning_status)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 1: Performed PUT Bucket versioning with status as %s on %s",
+                    versioning_status, self.bucket_name)
+        LOGGER.info("Step 2: Upload object %s with version %s bucket %s", self.object_name,
+                    versioning_status, self.bucket_name)
+        if versioning_status == "Suspended":
+            s3_cmn_lib.upload_version(self.s3_test_obj, bucket_name=self.bucket_name,
+                                      object_name=self.object_name, file_path=self.file_path,
+                                      versions_dict=self.versions, chk_null_version=True)
+        else:
+            s3_cmn_lib.upload_version(self.s3_test_obj, bucket_name=self.bucket_name,
+                                      object_name=self.object_name, file_path=self.file_path,
+                                      versions_dict=self.versions)
+        latest_ver = self.versions[self.object_name]["version_history"][-1]
+        LOGGER.info("Step 2: Successfully uploaded object %s to versioned bucket %s with "
+                    "version ID %s", self.object_name, self.bucket_name, latest_ver)
+        LOGGER.info("Step 3: Perform PUT Object Tagging for %s with 256 char tag value",
+                    self.object_name)
+        resp = s3_cmn_lib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
```

Comment:
Let's update the call to pass version_id here, as we are testing the limits when PUT Object Tagging is performed with version_id specified.